src/vs/base/browser/ui/highlightedlabel/highlightedLabel.ts

```diff
@@ -93,48 +105,80 @@ export class HighlightedLabel extends Disposable {
 		this.render();
 	}
 
+	private renderedText?: string;
+	private currentHighlightRanges: StaticRange[] = [];
```

Comment:
Is this variable redundant with labelHighlights? If it contains the exact same set of ranges, then you could get rid of this variable and then clearHighlights() can be just defined as labelHighlights.clear()